frontend/degree-plan/components/Requirements/QObject.tsx

```diff
@@ -72,14 +72,18 @@ const Attributes = ({ attributes }: { attributes: string[] }) => {
     </AttributeWrapper>
 }
 
-const SearchConditionWrapper = styled(BaseCourseContainer)`
+const SearchConditionWrapper = styled(BaseCourseContainer)<{$isSearched: boolean}>`
     display: flex;
     flex-wrap: wrap;
     gap: .5rem;
     background-color: var(--primary-color-light);
     box-shadow: 0px 0px 14px 2px rgba(0, 0, 0, 0.05);
     cursor: pointer;
     padding: .5rem .75rem;
+    ${props => !!props.$isSearched && `
+        border-radius: 10px;
+        box-shadow: 8px 6px 10px 8px #00000026;
+    `}
```

Comment:
I think this shadow looks a little weird bc it's so strong. Maybe we could change the color or apply some other effect to the surface of the search condition when clicked? let me know what you think

Reply:
I changed this to lighter and less aggressive shadow, similar to the one shown for droppable boxes